### PR TITLE
WIP - Managed services

### PR DESCRIPTION
**Pull request by sedroche**
@maleck13 

# THIS PR IS A WIP FOR A SPIKE. DO NOT REVIEW OR TRY TO TEST OR MERGE.

Accompanying card: https://trello.com/c/z5Fg3E7R/219-spike-managed-services-namespace-setup

### CODE REVIEW

File: evals/roles/mscontroller/defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+mscontroller_namespace: managed-service-admin
+mscontroller_roles: https://raw.githubusercontent.com/integr8ly/managed-service-controller/managed-services-admin-v0.0.1/deploy/rbac.yaml
```
Comment (maleck13) [inline evals/roles/mscontroller/defaults/main.yml]: is it possibly to go from a tag here? even if it is 0.0.1

File: evals/roles/olmcatalog/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+- name: Remove any left over resources to ensure clean install
+  import_tasks: uninstall.yml
```
Comment (pmccarthy) [inline evals/roles/olmcatalog/tasks/main.yml]: Similar to previous comment made around mscontroller role

File: evals/roles/mscontroller/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+- name: Remove any left over resources to ensure clean install
+  import_tasks: uninstall.yml
```
[the remark this answers is not in the document]
Comment (sedroche) [reply evals/roles/mscontroller/tasks/main.yml]: Although the installer should be a one shot thing that is not the case in reality. We may need to rerun the installer if things time out or whatever so we do need these to be idempotent.
Elsewhere in the playbooks we do checks to see if resources exist and install if they don't (so some checks are there)
I think it is easier to just ensure the resources are not there and then create them.

[the remark this answers is not in the document]
Comment (pmccarthy) [reply evals/roles/mscontroller/tasks/main.yml]: Fully agree that we may need to re-run the installer if it fails for whatever reason. My hope would be that if this happened and things need to be cleaned up we should run the uninstall playbook which would call on the roles uninstall task

[the remark this answers is not in the document]
Comment (sedroche) [reply evals/roles/mscontroller/tasks/main.yml]: I will add this to the uninstaller and refine the checks on the deployment. Thanks @pmccarthy

**Comment by camilamacedo86** [comment]
@sedroche is this PR still valid/needed? If yes, please, could you rebase with the master and in the first comment add all information as described in the [Pull Request Template](https://github.com/integr8ly/installation/blob/master/.github/PULL_REQUEST_TEMPLATE.md)? In this way, we can help here you move forward by testing it for example. Also, if being required came back to check the changes merged we will have the history and the info required to understand what was done here. Wdyt?

**Comment by camilamacedo86** [comment]
Update the title with WIP as its standard in order to make clear that it is a ongoing working

**Pull request closed without merging**